### PR TITLE
Cafcass people upgrade

### PR DESCRIPTION
**Pull request by yogendraU**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] the `keep-helm` label has been added, if the helm release should be persisted after a successful build

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by hmcts-shajahan — APPROVED** [review]

File: src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CaseDataServiceTest.java [hunk 1]
```diff
@@ -72,6 +73,7 @@ public void setUp() {
         when(authTokenGenerator.generate()).thenReturn(s2sToken);
     }
 
+    @Ignore
```
Comment (AlokDatta) [inline src/test/java/uk/gov/hmcts/reform/prl/services/cafcass/CaseDataServiceTest.java:76]: This is throwing code smell - https://sonarcloud.io/project/issues?resolved=false&sinceLeakPeriod=true&types=CODE_SMELL&pullRequest=1645&id=prl-cos-api&open=AYoc8tfAnseCIXeqSRX5

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/OtherPersonInTheCase.java [hunk 2]
```diff
@@ -8,18 +8,16 @@
...
-import java.util.List;
 
 @Data
 @AllArgsConstructor
 @NoArgsConstructor
 @JsonIgnoreProperties(ignoreUnknown = true)
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Builder(toBuilder = true)
-public class OtherPersonInTheCase {
+public class  OtherPersonInTheCase {
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/OtherPersonInTheCase.java:20]: remove the extra space.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+@Getter
+@RequiredArgsConstructor
+public enum PartyTypeEnum {
+
+
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java:11]: remove extra line gap

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    APPLICANT("APPLICANT", "Applicant"),
+    @JsonProperty("RESPONDENT")
+    RESPONDENT("RESPONDENT", "Respondent"),
+    @JsonProperty("OtherPeople")
+    OTHERPEOPLE("OTHERPEOPLE", "Other People");
+
+    private final String id;
+    private final String displayedValue;
+
+
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java:22]: remove extra line gap

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToParties.java [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Builder(toBuilder = true)
+public class RelationshipToParties {
+
+    private final String partyId;
+    private final String partyFullName;
+
+    private PartyTypeEnum partyType;
+
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToParties.java:22]: do we need this line gaps?

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    private PartyTypeEnum partyType;
+
+    private String childId;
+    private String childFullName;
+    private RelationshipsEnum relationType;
+    private String otherRelationDetails;
+    private YesOrNo childLivesWith;
+
+
+
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java:32]: extra line gaps, remove them please

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    private RelationshipsEnum childAndApplicantRelation;
+    private String childAndApplicantRelationOtherDetails;
+    private String applicantId;
+
+    private String respondentFullName;
+    private RelationshipsEnum childAndRespondentRelation;
+    private String childAndRespondentRelationOtherDetails;
+    private String respondentId;
+
+
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java:43]: extra line gaps, remove them please.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java (hunk 6 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java]: Fixed it thanks.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java (hunk 7 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToPartiesCafcass.java]: Fixed it thanks.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToParties.java (hunk 5 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/RelationshipToParties.java]: Fixed it thanks.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java (hunk 3 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/PartyTypeEnum.java]: Fixed it thanks.

File: src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/OtherPersonInTheCase.java (hunk 2 above)
Comment (yogendraU) [reply src/main/java/uk/gov/hmcts/reform/prl/models/dto/cafcass/OtherPersonInTheCase.java]: Fixed it thanks.

**Pull request merged**